frontend/src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,40 @@
+import { Wrapper, TextLogo, LogoStyle } from './Logo.styled';
+import logoImage from 'assets/images/taskManagerLogo.png';
+
+type LogoProps = {
+  hasClassName?: string;
+  hasAlt?: string;
```

Comment:
tutaj mógłby być po prostu `alt`
Ale raczej powinniśmy unikać stosowania skrótów.
Pytanie jeszcze, czy nie bedzie to wartość stała? Potrzebujey jej w propsach?

Reply:
usunięte